CorsixTH/Lua/game_ui.lua

```diff
@@ -600,6 +601,38 @@ function GameUI:onMouseUp(code, x, y)
   return UI.onMouseUp(self, code, x, y)
 end
 
+function GameUI:onMultiGesture(numfingers, dTheta, dDist, x, y)
+  -- only deal with 2 finger events for now
+  if numfingers == 2 then
+    -- calculate magnitude of pinch
+    local mag = math.abs(dDist)
+    if mag > 0.002 then
```

Comment:
Make this a variable (constant) defined near the top of the class. Let's avoid magic numbers.